### DISCUSSION TITLE
安装报错：Command not found: Command not found rmr

### DISCUSSION THREAD

**Discussion by ZhangZangQian** [apache/dolphinscheduler#13614] [Q&A] [answered]
安装报错，3.1.3版本，Zookeeper 3.8.1 版本

15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:java.library.path=/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:java.io.tmpdir=/tmp
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:java.compiler=<NA>
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.name=Linux
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.arch=amd64
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.version=3.10.0-1160.el7.x86_64
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:user.name=dolphinscheduler
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:user.home=/home/dolphinscheduler
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:user.dir=/opt/apache-dolphinscheduler-3.1.3-bin
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.memory.free=939MB
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.memory.max=989MB
15:09:38.567 [main] INFO org.apache.zookeeper.ZooKeeper - Client environment:os.memory.total=989MB
15:09:38.571 [main] INFO org.apache.zookeeper.ZooKeeper - Initiating client connection, connectString=localhost:2181 sessionTimeout=30000 watcher=org.apache.zookeeper.ZooKeeperMain$MyWatcher@7ec7ffd3
15:09:38.577 [main] INFO org.apache.zookeeper.common.X509Util - Setting -D jdk.tls.rejectClientInitiatedRenegotiation=true to disable client-initiated TLS renegotiation
15:09:38.582 [main] INFO org.apache.zookeeper.ClientCnxnSocket - jute.maxbuffer value is 1048575 Bytes
15:09:38.593 [main] INFO org.apache.zookeeper.ClientCnxn - zookeeper.request.timeout value is 0. feature enabled=false
ZooKeeper -server host:port -client-configuration properties-file cmd args
	addWatch [-m mode] path # optional mode is one of [PERSISTENT, PERSISTENT_RECURSIVE] - default is PERSISTENT_RECURSIVE
	addauth scheme auth
	close 
	config [-c] [-w] [-s]
	connect host:port
	create [-s] [-e] [-c] [-t ttl] path [data] [acl]
	delete [-v version] path
	deleteall path [-b batch size]
	delquota [-n|-b|-N|-B] path
	get [-s] [-w] path
	getAcl [-s] path
	getAllChildrenNumber path
	getEphemerals path
	history 
	listquota path
	ls [-s] [-w] [-R] path
	printwatches on|off
	quit 
	reconfig [-s] [-v version] [[-file path] | [-members serverID=host:port1:port2;port3[,...]*]] | [-add serverId=host:port1:port2;port3[,...]]* [-remove serverId[,...]*]
	redo cmdno
	removewatches path [-c|-d|-a] [-l]
	set [-s] [-v version] path data
	setAcl [-s] [-v version] [-R] path acl
	setquota -n|-b|-N|-B val path
	stat [-w] path
	sync path
	version 
	whoami 
Command not found: Command not found rmr
15:09:38.599 [main] ERROR org.apache.zookeeper.util.ServiceUtils - Exiting JVM with code 127
5.startup

**Comment by trunckman** [comment, marked as answer]
rmr 是zk3.5之前的命令了，后面改成deleteall了，所以用3.8版本zk需要修改remove-zk-node.sh这个脚本里的命令
